src/webhook/mutation/pod_mutator.go

```diff
@@ -794,3 +800,24 @@ func silentErrorResponse(podName string, err error) admission.Response {
 	rsp.Result.Message = fmt.Sprintf("Failed to inject into pod: %s because %s", podName, err.Error())
 	return rsp
 }
+
+// getSecretConfigData fetches the SecretConfig (initSecret.Config) from the pod namespace
+func (m *podMutator) getSecretConfigData(nsName string) (*standalone.SecretConfig, error) {
+	var scs corev1.Secret
```

Comment:
```suggestion
	var configSecret corev1.Secret
```

Reply:
Missed this in the first commit, as the msg was hidden.

Fixed in latest commit.